spark/src/test/java/com/nflabs/zeppelin/spark/SparkInterpreterTest.java

```diff
@@ -22,6 +22,7 @@ public void setUp() throws Exception {
 		Properties p = new Properties();
 		p.put("share", new HashMap<String, Object>());
 	    repl = SparkInterpreter.singleton();
+	    
```

Comment:
Are those changes important for migration to Gradle? May be we could drop them to simplify the life of those who review this PR, and address later in separate one